stylus-proc/tests/fail/public/constructor.rs

```diff
@@ -28,12 +28,8 @@ impl Contract {
     fn constructor() {}
 
     // error: constructor function can only be defined using the corresponding attribute
-    fn stylus_constructor() {}
-
-    // error: constructor function can only be defined using the corresponding attribute
-    #[selector(name = "stylusConstructor")]
+    #[selector(name = "stylus_constructor")]
```

Comment:
The selector passed here won't be converted to camelCase, so this won't work as intended.